### ISSUE TITLE
Build failed when verifying the setup

### ISSUE THREAD

**Issue by Punpun1643** [reposense/RepoSense#1782]
Hi! I am trying to set up the project. However, while verifying my setup, I face build failure issue below:

**Tell us about your environment**

**RepoSense Version:** `master` branch ([my forked repo](https://github.com/Punpun1643/RepoSense.git))
**OS and Version:** macOS Monterey 12.2.1

**Please include the steps to reproduce the bug.**
1. `./gradlew clean build`
2. `./gradlew test systemtest`

**What was expected to happen?**
Both finish with a `BUILD SUCCESSFUL` message.

**What actually happened? Please include a screenshot of the output.**
Both finish with a `BUILD FAILED` message, with `> Task :installFrontend FAILED` 

![94F12D68-5AB1-40FE-998A-5E4F050BF2C1_4_5005_c](https://user-images.githubusercontent.com/60144099/172039136-46d1e2e8-c71a-4d79-9abe-a4b1b189b41e.jpeg)

There seem to be many deprecated packages and there is a message stating that `Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0`

![0F662134-53EA-447D-BE39-23FB43FC789D_4_5005_c](https://user-images.githubusercontent.com/60144099/172039246-dd3c4479-2eb2-4e00-a6b9-f749c1c8d20c.jpeg)
![91EEF83A-28D7-4F4D-B234-72E9F3C66C20_4_5005_c](https://user-images.githubusercontent.com/60144099/172039248-5fa281e0-3be7-4639-91f7-935834ef3780.jpeg)
![07D7CC08-94EC-41EA-8CD0-81B34E0E9C73_4_5005_c](https://user-images.githubusercontent.com/60144099/172039312-1c5c3c0e-9cfd-4230-99f4-afc5c25b9b90.jpeg)

I found a similar issue https://github.com/reposense/RepoSense/issues/1133 as I was trying to resolve this issue. 
Based on the comments, I have tried the following:
1. Running `RepoConfigurationTest.java`, all test cases passed.
2. Running the app from code `gradlew run`, failed with the above-mentioned errors (failed for task `:installFrontend`). Report is generated in the `reposense-report` folder.
3. Generating report locally, report is generated successfully.

Based on this would it be alright for me to proceed to the next step? Thank you!

**Comment by yhtMinceraft1010X** [comment]
Thanks for opening this issue. There are already PRs available for some of what is mentioned:
* #1762 - Update of node packages.
* #1780 - Upgrade to Gradle 7.

As for the `installFrontend` failure, I am not sure what is happening at the moment.

**Comment by Punpun1643** [comment]
Hi @yhtMinceraft1010X, thank you for the response!

I tried doing the necessary updates to match both the PRs mentioned. Link to the updates could be found in [this branch](https://github.com/Punpun1643/RepoSense/tree/fix-setup) 
I think the Gradle upgrade seems to work. However, I am still facing issues on the task `:installFrontend` even after updating the node packages as per https://github.com/reposense/RepoSense/pull/1762

After updating the node packages, I made sure I did the following:
1.  `npm update` in the cypress directory
2.  Delete the `package-lock.json` file and `node_modules` directory and run `npm install` in the `frontend` directory
3.  Re-run `./gradlew clean build` at the project root.

These are the errors that I am still facing:
![FF52E098-D6E1-4F42-BD39-E75AC51F43C5](https://user-images.githubusercontent.com/60144099/172203861-5cdebfa1-75ec-47b5-9a8f-ebbda07940de.jpeg)

Is there any advice on how I should proceed? Thank you!
